dragonfly/block/kelp.go

```diff
@@ -0,0 +1,128 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/dragonfly/entity/physics"
+	"github.com/df-mc/dragonfly/dragonfly/item"
+	"github.com/df-mc/dragonfly/dragonfly/world"
+	"github.com/go-gl/mathgl/mgl64"
+	"math/rand"
+	"time"
+)
+
+type Kelp struct {
+
+	//0-15, defines the age of the kelp block. If age is 15, kelp won't grow any further.
```

Comment:
Make sure to start documentation with the name of the field/method/type and add a space after the `//`.